src/pages/Inventory.page.js

```diff
@@ -5,11 +5,22 @@ export class InventoryPage extends BaseSwagLabPage {
 
     headerTitle = this.page.locator('.title');
 
+    sortingButton = this.page.locator('[data-test="product-sort-container"]');
+
+    sortingOptionList = this.page.locator('select option');
+
     inventoryItems = this.page.locator('.inventory_item');
 
+    inventoryPrices = this.page.locator('[data-test="inventory-item-price"]');
+    
     addItemToCartButton = this.page.locator('[id^="add-to-cart"]');
-
+ 
     async addItemToCartById(id) {
         await this.addItemToCartButton.nth(id).click();
     }
+
+    async getElementContent(elements) {
+        return await elements.allTextContents(); 
```

Comment:
There is no sence in wrapping just oner function in another. We still can use allTextContents() instead of this function and we should